pkg/server/metrics/metrics.go

```diff
@@ -120,16 +112,25 @@ func newServerMetrics() *ServerMetrics {
 		prometheus.GaugeOpts{
 			Namespace: Namespace,
 			Subsystem: Subsystem,
-			Name:      PendingDialsMetric,
+			Name:      "pending_backend_dials",
 			Help:      "Current number of pending backend dial requests",
 		},
 		[]string{},
 	)
+	establishedConns := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: Namespace,
+			Subsystem: Subsystem,
+			Name:      "established_connections",
+			Help:      "Current number of established connections (post-dial).",
```

Comment:
nit: can we add "end-to-end" in the description?
```suggestion
			Help:      "Current number of established end-to-end connections (post-dial).",
```

Reply:
Done.